Snakefile

```diff
@@ -163,7 +166,12 @@ rule prepare_sector_network:
         ),
         pipelines="resources/custom_data/pipelines.csv"
         if config["custom_data"]["gas_network"]
-        else "resources/gas_networks/gas_network_elec_s{simpl}_{clusters}.csv",
+        else "resources/gas_networks/gas_network_elec_s{simpl}_{clusters}.csv"
+        if not (
+            config["sector"]["hydrogen"]["network_routes"] == "greenfield"
+            and not config["sector"]["hydrogen"]["gas_network_repurposing"]
+        )
+        else "data/pipelines.csv",
```

Comment:
This doesnt' look so clean to me, and i'm unsure about the need for that. I drop few comments:
- since there is already the custom_data/pipelines, is there the need to add data/pipelines as well?
- are the conditions (the if/else cases) expected to grow in the future? if so, an auxiliary function may help

From a styling point of you, if these conditions are needed, I feel we can either we use if ... elif ... else ... , or we create an auxiliary function in the snakefile that takes care of the switch among cases.

What do you think?

Reply:
In the case (config["sector"]["hydrogen"]["network_routes"] == "greenfield" and not config["sector"]["hydrogen"]["gas_network_repurposing"]), we dont go into the rule prepare_gas_network, but also dont have any custom gas_network data. As we do h2_links = pd.read_csv(snakemake.input.pipelines) for all those cases  in prepare_sector_network.py, we need some kind of empty file with the right columns for this case. It would also be okay to read this file out of the filepath custom_data, but i thought it might be an issue, if someone has custom_data, but wants to model without it for benchmarking. 

Guess the conditions doesnt grow any more in the future in this case 

What do you think, how should we handle it?

Reply:
@Eddy-JV could you please share your thoughts here?

Reply:
We could also create an empty dataframe with the right columns in prepare_sector_network.py for the case (config["sector"]["hydrogen"]["network_routes"] == "greenfield" and not config["sector"]["hydrogen"]["gas_network_repurposing"]) and reading the file for all other cases, just say what you prefer :)

Reply:
Hi everyone, I updated the PR #283 inlcuding the comment of @davide-f and solving the problem faced by @doneachh . Please check and provide me with any feedback.

This way, this PR can be disregarded.